tests/Unit/Core/Cart/AbstractCartTest.php

```diff
@@ -233,6 +257,20 @@ protected function getProductFromFixtureId($productFixtureId)
         return null;
     }
 
+    /**
+     * @param int $combinationFixtureName fixture combination name
+     *
+     * @return Combination|null
+     */
+    protected function getCombinationFromFixtureName($combinationFixtureName)
+    {
+        if (isset($this->combinations[$combinationFixtureName])) {
+            return $this->combinations[$combinationFixtureName];
+        }
+
+        return null;
```

Comment:
Is it really useful?

Reply:
I think it is, since the array containing fixtures is only a cache, we could want to change the storage of that